devito/operator/registry.py

```diff
@@ -26,7 +26,7 @@ class OperatorRegistry(OrderedDict, metaclass=Singleton):
     """
 
     _modes = ('noop', 'advanced', 'advanced-fsg')
-    _languages = ('C', 'openmp', 'openacc', 'cuda')
+    _languages = ('C', 'openmp', 'openacc', 'cuda', 'hip')
```

Comment:
So what does it actually do if you use this without pro? Just the standard cpup code but compiled with HIP?

Reply:
no you will get an InvalidOperator exception as soon as you try to build an Operaator